src/main/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJob.java

```diff
@@ -171,4 +169,14 @@ public ResultSupplier getResultSupplier() {
             metaData -> uploadFiles()
         ));
     }
+
+    private void updateDateSent(File file) {
+        databaseService.execute(getDatabaseConfig(), connection -> {
+            try {
+                databaseService.executeUpdate(connection, UPDATE_SQL_QUERY, file.getName(), fileType);
+            } catch (SQLException e) {
+                log.error(fileType + ": Failed to update file: " + file.getName(), e);
```

Comment:
Could the log message be more clear?

For example fileType + ": failed to mark document: " + file.getName + " as uploaded"

Reply:
Done